tests/phpunit/integration/api/filters/beansAddFilter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for beans_add_filter()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Filters
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Filters;
+
+use Beans\Framework\Tests\Integration\API\Filters\Includes\Filters_Test_Case;
+
+require_once __DIR__ . '/includes/class-filters-test-case.php';
+
+/**
+ * Class Tests_BeansAddFilter
+ *
+ * @package Beans\Framework\Tests\Integration\API\Filters
+ * @group   integration-tests
+ * @group   api
+ */
+class Tests_BeansAddFilter extends Filters_Test_Case {
+
+	/**
+	 * Test beans_add_filter() should add (register) the filter when callback is callable.
+	 */
+	public function test_should_add_the_filter_when_callable() {
+
+		foreach ( static::$test_filters as $beans_id => $filter ) {
+
+			if ( ! isset( $filter['callback'] ) ) {
+				continue;
+			}
+
+			// Test that the filter has not yet been added.
+			$this->assertFalse( has_filter( $filter['hook'], $filter['callback'] ) );
+
+			// Let's add it.
+			$this->assertTrue( beans_add_filter( $filter['hook'], $filter['callback'], $filter['priority'], $filter['args'] ) );
+
+			// Check that the filter was registered.
+			$this->assertTrue( has_filter( $filter['hook'], $filter['callback'] ) !== false );
+
+			remove_filter( $filter['hook'], $filter['callback'], $filter['priority'] );
```

Comment:
Should the "Clean up." comment (like in beansAddAnonymousFilter.php)  be added here and in the next test?